src/LibSDL2.jl

```diff
@@ -7098,6 +7171,10 @@ const SDL_MUTEX_TIMEDOUT = 1
 
 const SDL_MUTEX_MAXWAIT = ~(Uint32(0))
 
+const SDL_beginthread = _beginthreadex
```

Comment:
This throws an error saying it doesn't exist. I'm not sure if I'm supposed to just delete it?

Reply:
This does not show up when I generate the bindings :sweat_smile: Maybe because we're using different package versions? I've updated everything.